torchvision/datasets/kinetics.py

```diff
@@ -20,44 +34,103 @@ class Kinetics400(VisionDataset):
     Note that we drop clips which do not have exactly ``frames_per_clip`` elements, so not all
     frames in a video might be present.
 
-    Internally, it uses a VideoClips object to handle clip creation.
-
     Args:
-        root (string): Root directory of the Kinetics-400 Dataset. Should be structured as follows:
-
+        root (string): Root directory of the Kinetics Dataset.
+            Directory should be structured as follows:
             .. code::
 
                 root/
-                ├── class1
-                │   ├── clip1.avi
-                │   ├── clip2.avi
-                │   └── ...
-                └── class2
-                    ├── clipx.avi
-                    └── ...
-
+                ├── split
+                │   ├──  class1
+                │   │   ├──  clip1.mp4
+                │   │   ├──  clip2.mp4
+                │   │   ├──  clip3.mp4
+                │   │   ├──  ...
+                │   ├──  class2
+                │   │   ├──   clipx.mp4
+                │   │    └── ...
+            Split is appended using the split argument.
+        num_classes (int): select between Kinetics-400, Kinetics-600, and Kinetics-700
+        split (str): split of the dataset to consider; currently supports ["train", "val"]
+        frame_rate (float): If not None, interpolate different frame rate for each clip.
         frames_per_clip (int): number of frames in a clip
         step_between_clips (int): number of frames between each clip
+        annotation_path (str): path to official Kinetics annotation file.
         transform (callable, optional): A function/transform that  takes in a TxHxWxC video
             and returns a transformed version.
+        download (bool): Download the official version of the dataset to root folder.
+        num_workers (int): Use multiple workers for VideoClips creation
+        _num_download_workers (int): Use multiprocessing in order to speed up download.
```

Comment:
Either make this public (remove the underscore) or remove it from the docstring. No hard opinion here, but since the download is quite lengthy I would make it public. Otherwise no one will know about this unless the dive into the source.